tests/gtfs2ntfs.rs

```diff
@@ -126,6 +126,16 @@ fn test_minimal_ziped_sub_dir_gtfs() {
     });
 }
 
+#[test]
+fn test_minimal_ziped_sub_dir_gtfs_with_hidden_files() {
+    test_in_tmp_dir(|path| {
+        let input = "./tests/fixtures/ziped_gtfs/sub_dir_gtfs_with_hidden_files.zip";
```

Comment:
your zip contains a folder "input" at the root. This is what you want?

Reply:
> your zip contains a folder "input" at the root. This is what you want?

Yes. A subfolder + another system subfolder with hidden files. A dirty GTFS  :hankey: